modules/components/pages/outputs/schema_registry.adoc

```diff
@@ -0,0 +1,276 @@
+= schema_registry
+:type: output
+:status: beta
+:categories: ["Integration"]
+
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Publishes schemas to a schema registry.
+
+Introduced in version 4.32.2.
+
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+output:
+  label: ""
+  schema_registry:
+    url: "" # No default (required)
+    subject: "" # No default (required)
+    max_in_flight: 64
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
```

Comment:
@Jeffail. Is there a specific use case for this connector? Can we describe it here?

Also, is this a community or certified connector? I have assumed certified.

Reply:
Both are enterprise, so on cloud they're certified (I think).